internal/cmd/operator-sdk/generate/internal/manifests.go

```diff
@@ -35,6 +36,19 @@ func GetManifestObjects(c *collector.Manifests) (objs []controllerutil.Object) {
 		objs = append(objs, &c.ServiceAccounts[i])
 	}
 
+	// All Services passed in should be written.
```

Comment:
This isn't quite true. OLM manages webhooks and their services, no explicitly defined webhook `Service` objects should be passed. You can filter out `Service`s by finding those associated with `{Mutating,Validating}WebhookConfiguration`s included in a bundled CSV (all webhook configs in the `collector.Manifests`).

Reply:
Do we just match on namespace and name? What if a service declares multiple ports and one of them is for the webhook?

Do you think we need to filter out webhook service ports and then filter out any services that have no ports?

Reply:
Can we constrain the problem by filtering any service that matches a webhook config by name/namespace, and document that webhooks need their own separate service? Modifying a resource's functional content doesn't feel right.

Reply:
We could. I think I have a working solution that does what I suggest above. Let me know what you think.